src/clusterfuzz/_internal/platforms/android/adb.py

```diff
@@ -338,6 +338,18 @@ def hard_reset():
   soft_reset_cmd = get_adb_command_line('reboot')
   execute_command(soft_reset_cmd, timeout=RECOVERY_CMD_TIMEOUT)
 
+  if environment.is_android_emulator():
+    #For recovery state
```

Comment:
Can you please format this comment a bit better by adding a space after the "#" and making it a more complete sentence? e.g. `# Handle recovery states for emulator by doing a complete wipe`.

Reply:
Would deleting the comment altogether also work? I was thinking the logging messages and is_android_emulator() call might adequately explain to a new reader what the submitted lines are doing.

Reply:
sounds good. the log messages seem self explanatory enough.